engine/src/deploy.cpp

```diff
@@ -973,19 +974,20 @@ void MCIdeExtract::exec_ctxt(MCExecContext& ctxt)
     if (!ctxt . EvalExprAsStringRef(m_filename, EE_IDE_EXTRACT_BADFILENAME, &t_filename))
         return;
 		
+	if (!ctxt . HasError())
```

Comment:
Its not even necessary at all - if there is an error at that point, then the function would have already returned!